README.md

```diff
@@ -0,0 +1,142 @@
+![imgix logo](https://assets.imgix.net/imgix-logo-web-2014.pdf?page=2&fm=png&w=200&h=200)
+
+[![Build Status](https://travis-ci.org/imgix/imgix-python.png?branch=master)](https://travis-ci.org/imgix/imgix-python)
+
+A Python client library for generating URLs with imgix. imgix is a
+high-performance distributed image processing service. More information
+can be found at <http://www.imgix.com>.
+
+Installation
+============
+
+``` {.bash}
+$ pip install imgix
+```
+
+Basic Usage
+===========
+
+To begin creating imgix URLs programmatically, simply import the imgix
+library and create a URL builder. The URL builder can be reused to
+create URLs for any images on the domains it is provided.
+
+``` {.python}
```

Comment:
Can we change all instances of code blocks to use `python` instead of `{.python}`? I would like to have it consistent with the rest of the SDK docs.

Same goes for any occurrences of `bash`, `html`, etc